.eslintrc.yml

```diff
@@ -9,6 +9,7 @@ rules:
   no-mixed-operators: 0
   class-methods-use-this: 0
   no-nested-ternary: 0
+  linebreak-style: 0
```

Comment:
I don't want to disable this rule, either set line endings to LF in your editor or try `$ eslint src/* --ext .js --fix`.

Reply:
https://github.com/kalkih/mini-media-player/pull/308/commits/07a4f6e6e173de834c5841704202486053db4571
used the eslint src/* --ext .js --fix, as set LF is the same but manually for all js files.